frontend/packages/dev-console/src/components/project-access/project-access-form-utils.ts

```diff
@@ -5,7 +5,22 @@ import {
   ProjectAccessRoles,
 } from './project-access-form-utils-types';
 
-export const filterRoleBindings = (roleBindings: RoleBinding[], roles): RoleBinding[] => {
+export const defaultAccessRoles = {
+  admin: 'Admin',
+  edit: 'Edit',
+  view: 'View',
+};
+
+export type Roles = {
+  [key: string]: string;
+};
+
+export const getAvailableAccessRoles = () => {
```

Comment:
Personal preferences is to provide types here, esp. with this JSON.parse

```suggestion
export const getAvailableAccessRoles = (): string[] | undefined => {
```